### PR TITLE
Changes for post install configuration

### PR DESCRIPTION
**Pull request by oneumyvakin**
TASK SIO-2396 Add License service
TASK SIO-2396 Add IP Blocks list
TASK SIO-2396 Add Compute Resource install step status error
TASK SIO-2396 Make IP Blocks and Location optionally at adding Compute Resource
TASK SIO-2396 Add WithLogger client option
TASK SIO-2396 Add Update method for User
TASK SIO-2396 Fix IsSnapshotsEnabled and add ImageFormat to Plan
TASK SIO-2396 Add Roles

### CODE REVIEW

File: computeResourcesInstallSteps.go [hunk 1]
```diff
@@ -6,6 +6,10 @@ import (
 	"fmt"
 )
 
+const (
```
Comment (dshemin) [inline computeResourcesInstallSteps.go:9]: Could you add `ComputeResourceInstallStepStatus` type, please.

**Review by dshemin — CHANGES_REQUESTED** [review]

File: computeResourcesInstallSteps.go [hunk 2]
```diff
@@ -6,17 +6,23 @@ import (
...
+	ComputeResourceInstallStepStatusError ComputeResourceInstallStepStatus = "error"
+)
+
+type ComputeResourceInstallStepStatus string
```
Comment (dshemin) [inline computeResourcesInstallSteps.go:13]: Please place `type` before constants. It's a common way to define enums in Golang.

**Review by dshemin — CHANGES_REQUESTED** [review]

**Review by dshemin — APPROVED** [review]
LGTM!

**Comment by oneumyvakin** [comment]
Thank you!

**Pull request merged**